### DISCUSSION TITLE
Viewports not being recognised

### DISCUSSION THREAD

**Discussion by noahfrn** [tbabej/taskwiki#334] [Community support]
Having an issue with viewports not being recognised, and tasks therefore not inheriting attributes.

Here's an example which isn't recognised on my machine: 

`== Research Project | project:ResearchProject ==`

Viewport inspection doesn't work either, and they aren't recognised in my vim-tagbar.

Any idea as to what might be going on ? As far as I can tell, all other functionality except for preset headers and viewports works fine.

**Comment by tbabej** [comment]
Perhaps you have `markdown` syntax turned on, which would require you to write your viewports like this?

```
## Research Project | project:ResearchProject
```

**Comment by noahfrn** [comment]
No I've got vimwiki syntax on, as is indicated by my airline bar.

Thanks for taking the time to answer !

**Comment by civrot** [comment]
Did you ever get a resolution on this @NoahFournier? I just installed this plug in and I am experiencing the same issue.

**Comment by civrot** [comment]
NVM, it was me. I was using two pipes for the filter. Once I fixed, ti works. Now I just need to stop that autofoling from happen that this plugin apparently enables!
